src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -85,4 +87,34 @@ export default class JsonRpcNetworkClient
     const accountLinesResponse: AccountLinesResponse = axiosResponse.data
     return accountLinesResponse
   }
+
+  /**
+   * Submits a gateway_balances request to the rippled JSON RPC.
+   * @see https://xrpl.org/gateway_balances.html
+   *
+   * @param account The XRPL account for which to retrieve balances.
+   * @param addressesToExclude (Optional) An operational address to exclude from the balances issued, or an array of such addresses.
+   * @see https://xrpl.org/issuing-and-operational-addresses.html
+   */
+  public async getGatewayBalances(
+    account: string,
+    addressesToExclude?: string | Array<string>,
```

Comment:
as above, can this just always be an array? it seems like it would simplify a lot of stuff

Reply:
yar